examples/carousel/ui/theme.slint

```diff
@@ -1,8 +1,8 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: MIT
 
-import "fonts/Roboto-Regular.ttf";
-import "fonts/Roboto-Bold.ttf";
+import "../fonts/Roboto-Regular.ttf";
+import "../fonts/Roboto-Bold.ttf";
```

Comment:
why this change?

Reply:
This change will be removed. I have trouble with relative file loading at the moment. With the fix this will be removed.